services/director-v2/src/simcore_service_director_v2/models/schemas/services.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+
+from ..constants import SERVICE_IMAGE_NAME_RE, VERSION_RE
+from .image_meta import ImageMetaData
+
+
+class ServiceType(str, Enum):
+    COMPUTATIONAL = "computational"
+    INTERACTIVE = "interactive"
```

Comment:
I feel that this was redefined. Maybe the prior one has a different usage? If so could you add a comment?